lib/payment.js

```diff
@@ -91,6 +103,8 @@ class Payment {
         _sender.set(this, sender);
         _recipient.set(this, recipient);
         _senderRef.set(this, senderRef);
+        if (undefined !== senderPayload && null !== senderPayload)
```

Comment:
Can be simplified to 'if (! senderPayload)'

Reply:
This depends on what to accept as payload. The current implementation allows for empty string, while your suggestion won't do that. So it all depends on the value space we need for sender payload.

Are there cases where empty string sender payload would make sense? Alternatively, should we also allow value `null`?

Reply:
I'll adjust slightly so that we, at least initially, skip the possibility of having empty string sender payload.